modules/installation-configuration-parameters.adoc

```diff
@@ -923,7 +923,8 @@ ifdef::rhv[]
 
 Additional {rh-virtualization} configuration parameters are described in the following table:
 
-.Additional {rh-virtualization} parameters for clusters
+[id=`Additional-virt-parameters-for-clusters`]
```

Comment:
> General comment to maybe clean up in a later PR: In trying to find where these modules were being built, I noticed there were quite a few references at the top of modules to:
> 
> ```
> // * installing/installing_rhv/installing-rhv-custom.adoc
> ```
> 
> which no longer seems to exist. Maybe it was an old file name for `installing-rhv-customizations.adoc`?

Yeah, I see what you're talking about. I don't know why these are here and commented out. While you're right that the file name here is incorrect, I see a whole lotta files commented out, which leads me to wonder why. I would prefer to leave this alone for now, because I think the proper way to address it is not simply to remove this one commend out module, but understand why all these modules are commented out and if necessary, remove them all.

Reply:
> Should be all lower-case and with double-quotes instead of backticks:
> 
> ```
> [id="additional-virt-parameters-for-clusters"]
> ```
> 
> This was also causing the Travis build to fail.

Roger that. Fixed. Is the all lower-case rule an OpenShift guideline? I'm not aware of it being a mod docs guideline.

Reply:
> While you're right that the file name here is incorrect, I see a whole lotta files commented out, which leads me to wonder why. I would prefer to leave this alone for now, because I think the proper way to address it is not simply to remove this one commend out module, but understand why all these modules are commented out and if necessary, remove them all.

These lines are for metadata in each module, listing what assemblies they've been included in: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

Reply:
> Roger that. Fixed. Is the all lower-case rule an OpenShift guideline? I'm not aware of it being a mod docs guideline.

I don't think the lowercase thing is in the OpenShift guidelines. But all of our IDs are all lowercase normally, so might as well sync it up.

However I forgot to mention in that comment that it was also missing the `_{context}` in the ID, so it will need that still, too.

Reply:
> These lines are for metadata in each module, listing what assemblies they've been included in: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#module-file-metadata
@adellape  D'oh! I should have seed that. Fixed in all places.

Reply:
> However I forgot to mention in that comment that it was also missing the `_{context}` in the ID, so it will need that still, too. (I realize it's just an ID for a table and not an actual section heading, but for the purposes of making sure there are never any duplicate IDs, the rule still seems valid.)

Fixed.